packages/web/src/pages/audio-rewards-page/components/modals/ChallengeRewardsModal/AudioMatchingRewardsModalContent.tsx

```diff
@@ -72,6 +87,45 @@ const ClaimInProgressSpinner = () => (
   <LoadingSpinner className={styles.spinner} />
 )
 
+/**
+ * Filters for only $AUDIO matching challenges and generates row
+ * items in correct format for SummaryTable. Also calculates the
+ * total claimable amount.
+ */
+const useUndisbursedChallengesCooldown = () => {
```

Comment:
I'm cool with it being a hook, but will eventually want to move it out of the component file so we can use it on mobile as well?

Reply:
Also wondering if there's a better name for this :sweat_smile: